src/figuresClasses.ts

```diff
@@ -1,19 +1,109 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
it will be good to use typing here
```suggestion
  shape: Figures;
  color: Color;
```